output/scripts/language_by_gender.js

```diff
@@ -0,0 +1,47 @@
+
+  if(typeof(Bokeh) !== "undefined") {
+    console.log("Bokeh: BokehJS loaded, going straight to plotting");
+    Bokeh.embed.inject_plot("e9ab7b57-9009-4939-b449-09342ab64e22", all_models);
+  } else {
+    load_lib(bokehjs_url, function() {
+      console.log("Bokeh: BokehJS plotting callback run at", new Date())
+      Bokeh.embed.inject_plot("e9ab7b57-9009-4939-b449-09342ab64e22", all_models);
+    });
+  }
+
+}(this));
```

Comment:
@vivekiitkgp remember we were having a discussion about how to generate files without new line the end of line, this is how we do it.